documentation/documentation/docs/evaluators/python_evaluator.md

```diff
@@ -0,0 +1,11 @@
+# Python evaluator
+## General usage
+This evaluator is responsible for running and executing tests on a student's Python code.
+
+## Structure
+When submitting the project a teacher can add a requirements manifest `filename of manifest`, this way only the packages in the requirements file are usable on the evaluator.
+
+When no manifest is present, students are able to install their own depedencies with a `requirements.txt` and a `dev-requirements.txt`.
+
+## Running tests
+When a `run_tests.sh` is present in the project assignment files, it will be run when the student is submitting their code.
```

Comment:
The root of a students submission will be `/submission`

Reply:
https://github.com/SELab-2/UGent-3/pull/250/commits/a175039634b28c72360c862b6f82fb2f88d1afe0